Sources/Persistence/ServerSyncJobStore.swift

```diff
@@ -0,0 +1,68 @@
+import Foundation
+
+struct ServerSyncJobStore {
+
+    private var jobStoreArray: [ServerSyncJob] = []
+    private let syncJobStoreQueue = DispatchQueue(label: "syncJobStoreQueue")
+
+    init() {
+        self.jobStoreArray = self.loadOperations()
+    }
+
+    private func loadOperations() -> [ServerSyncJob] {
+        guard let operations = NSData(contentsOfFile: "syncJobStoreQueue") else {
+            return []
+        }
+
+        let jsonDecoder = JSONDecoder()
+        guard let operationsArray = try? jsonDecoder.decode([ServerSyncJob].self, from: (operations as Data)) else {
+            return []
```

Comment:
Do we want to print out a warning message here?

"Failed to load previously stored operations, continuing without them"